src/status_im/ui2/screens/chat/messages/message.cljs

```diff
@@ -128,13 +128,14 @@
           (:parsed-text content)))
 
 (defn quoted-message
-  [{:keys [message-id chat-id]} reply pin?]
```

Comment:
If we are not using reply, can we just remove it from params

Reply:
Removed it, Thanks Parvesh